AmplifyPlugins/Storage/Tests/AWSS3StoragePluginTests/Mocks/MockAWSS3PreSignedURLBuilder.swift

```diff
@@ -22,9 +22,15 @@ extension MockAWSS3PreSignedURLBuilder: AWSS3PreSignedURLBuilderBehavior {
     func getPreSignedURL(
         key: String,
         signingOperation: AWSS3SigningOperation,
+        metadata: [String : String]?,
         accelerate: Bool?,
         expires: Int64?) async throws -> URL {
-            interactions.append("\(#function) \(key) \(signingOperation) \(String(describing: expires))")
+            if let metadata = metadata {
```

Comment:
nit: we can remove if block and add 
`interactions.append("\(#function) \(key) \(signingOperation) \(String(describing: metadata)) \(String(describing: expires))")`

or 

`if let metdata = metadata, expires = expires { }`

Reply:
removed if block per suggestion